src/main/java/nextstep/courses/domain/Students.java

```diff
@@ -2,19 +2,30 @@
 
 import nextstep.users.domain.NsUser;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 public class Students {
     private static final int MAX_STUDENTS = 20;
     private static final String MAX_STUDENT_EXCEPTION = "최대 인원을 초과하였습니다.";
     private static final String ALREADY_STUDENT = "이미 등록된 학생 입니다.";
     private static final String NOT_INFO = "등록된 학생 정보가 없습니다.";
     private static final String NULL_EXCEPTION = "학생 데이터가 잘못 생성되었습니다.";
-    private final Map<Long, NsUser> studentsMap = new HashMap<>();
+    private Map<Long, NsUser> studentsMap = new HashMap<>();
+
+    private List<NsUser> users = new ArrayList<>();
```

Comment:
```suggestion
    private List<SessionUserMapping> users = new ArrayList<>();
```
수강신청한 학생의 목록은 SessionUserMapping을 가지는 것은 어떨까?